Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/HDInsightHelper.java

```diff
@@ -36,7 +37,7 @@
     boolean isOptIn();
 
     @Nullable
-    default NodeActionListener createAddNewHDInsightReaderClusterAction(@NotNull HDInsightRootModule module, @NotNull String clusterName) {
+    default NodeActionListener createAddNewHDInsightReaderClusterAction(@NotNull HDInsightRootModule module, @NotNull ClusterDetail clusterDetail) {
```

Comment:
How about the eclipse side effect?

Reply:
This action is designed for HDI reader cluster link action and is only called by IntelliJ. Therefore, no side effect for Eclipse.